snapshots/devmapper/snapshotter.go

```diff
@@ -428,17 +481,21 @@ func (s *Snapshotter) getDevicePath(snap storage.Snapshot) string {
 	return dmsetup.GetFullDevicePath(name)
 }
 
-func (s *Snapshotter) buildMounts(snap storage.Snapshot) []mount.Mount {
+func (s *Snapshotter) buildMounts(snap storage.Snapshot, fileSystemType fsType) []mount.Mount {
```

Comment:
Since buildMounts() is not exported, I'd like to pass a context.Context rather than making a brand-new context.

Reply:
Makes sense. Modified buildMount to have ctx passed on.